src/pages/ProfilePage/ReviewPage.tsx

```diff
@@ -16,15 +16,11 @@ interface ReviewTypes {
 }
 
 const ReviewPage = () => {
-  const [selectedButton, setSelectedButton] = useState('mate')
+  const [selectedButton, setSelectedButton] = useState('1')
```

Comment:
매직 리터럴, 매직 넘버는 제거해주시면 좋습니다!

```suggestion
  const GOODS_REVIEW = '1';
  const MATE_REVIEW = '2';
  
  const [selectedButton, setSelectedButton] = useState(GOODS_REVIEW)
```